### ISSUE TITLE
Is onshot125 competible with the scrpit,Since there's no enough aux chennel

### ISSUE THREAD

**Issue by JiangNanGenius** [IamPete1/Lynchpin#4]
I stucked there I only have 3 aux channels with my newly bought cuav V5 nano.

**Comment by JiangNanGenius** [comment]
still trying but there seems to be no pwm outputs

**Comment by IamPete1** [comment]
Yeah, your out of luck there, only 3 capable D-Shot outputs, it should work on the main PWM outs (I have not tested). You wont get BLHeli telem. You will have to shuffle the escs through to set them up as reversible 3D with BLHeli suite.

**Comment by JiangNanGenius** [comment]
> Yeah, your out of luck there, only 3 capable D-Shot outputs, it should work on the main PWM outs (I have not tested). You wont get BLHeli telem. You will have to shuffle the escs through to set them up as reversible 3D with BLHeli suite.

I found that it works, and I finished my first flight it seems to be ok. The only thing is that I have to set blheli through the aux channel. And another thing is I found that you didn't mention how to let the two sliders functioning in the mission planner. And then I just confused about that since the DSM only provides 8 channels with one satellite receiver, so that I can't set my channel to yours and copy your configuration.
![Lynchpin](https://user-images.githubusercontent.com/79638157/109498979-3c18a880-7acf-11eb-914e-a23cbc4fa307.jpg)
